File: src/main/java/com/dotgoing/repository/PersonRepository.java
package com.dotgoing.repository;

import com.dotgoing.model.Person;
import org.springframework.data.repository.PagingAndSortingRepository;
import java.util.List;
import org.springframework.data.repository.query.Param;

public interface PersonRepository extends PagingAndSortingRepository<Person, Long> {

    List<Person> findByLastName(@Param("name") String name);

}

File: src/main/kotlin/com/dotgoing/repository/PersonRepository.kt
package com.dotgoing.repository

import com.dotgoing.model.User
import org.springframework.data.repository.PagingAndSortingRepository
import org.springframework.data.repository.query.Param

interface PersonRepository : PagingAndSortingRepository<User, Long> {
    fun findByLastName(@Param("name") name: String): List<User>
}